### ISSUE TITLE
CFG for training

### ISSUE THREAD

**Issue by zhangdan8962** [facebookresearch/DiT#51]
Hi, I am wondering where did you utilize the CFG during training? And what is the value for label dropout?

**Comment by Lishuo-BIT** [comment]
According to the shared training code, I don't think they used CFG during training.
